src/page-objects/storefront/Category.ts

```diff
@@ -13,6 +13,6 @@ export class Category implements PageObject {
     }
 
     url(categoryName: string): string {
-        return `${categoryName}`;
+        return `${categoryName}/`;
```

Comment:
and here? Why is there a `/`

Reply:
weird bug in 6.5, without the `/` it does work, should work both ways in 6.6